azerothcore/azerothcore-wotlk
# fix (db): Thrash

Thrash was messed up extremely with a ton of npcs, this is a bit of a revert dealing with those npcs but with corrections to ensure the spell is not spammed and triggers only when they are suppose to.



## Changes Proposed:
-  Fix thrash on affected NPCS

## Issues Addressed:

- Closes I am pretty sure there is more then a few issues mentioned

## SOURCE:


## Tests Performed:

- Builds and Performs
- Affected rows: 318  Found rows: 0  Warnings: 0  Duration for 157 queries: 0.140 sec. 


## How to Test the Changes:


1. Go piss off a npc with thrash
2. Notice how they are not Excessively Over Power on their damange
3. Notice how they no longer have Excessively huge range of physical attack
4. May be better if you root then and step away when they have thrash to ensure the physical damage range is corrected

## Known Issues and TODO List:


- [x] Do alot of sugar booger after this fubar clean up
- [x] Pray to Cthulhu for the world to end


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
